### PR TITLE
ktest: add grub-reboot cheat sheet

### PR DESCRIPTION
**Pull request by marc-hb**
I found all `grub-reboot` guides either too long or too short or confusing
or all of the above :-)

Signed-off-by: Marc Herbert <marc.herbert@intel.com>

### CODE REVIEW

File: getting_started/setup/setup_ktest_environment.rst [hunk 1]
```diff
@@ -59,15 +59,71 @@ working kernel if your changes fail to boot.
 
    # Use your text editor of choice.
    sudo emacs /etc/default/grub
+   sudo update-grub
```
Comment (fredoh9) [inline getting_started/setup/setup_ktest_environment.rst:62]: Should we put what we edit in /etc/default/grub in between?

**Review by fredoh9 — COMMENTED** [review]
Thanks Marc, I liked the details about grub-reboot. This will be the bible of ktest.

File: getting_started/setup/setup_ktest_environment.rst (hunk 1 above)
Comment (marc-hb) [reply getting_started/setup/setup_ktest_environment.rst:62]: Logically yes but that would consume two  `code-block`s for two lines, a bit too verbose IMHO. I'm assuming the reader's look-ahead buffer is at least 2 lines long...

**Review by deb-intel — APPROVED** [review]
Nicely worded!

**Pull request merged**